--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -18,7 +18,7 @@
 
 find_package(cetmodules 3.08.00 REQUIRED)
 
-project(artdaq-database VERSION 3.03.00)
+project(artdaq-database VERSION 3.04.00)
 
 set(TOP_CMAKE_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 set(TOP_CMAKE_BINARY_DIR ${CMAKE_CURRENT_BINARY_DIR})
--- a/artdaq-database/SWIGBindings/python/PythonAddon.cmake
+++ b/artdaq-database/SWIGBindings/python/PythonAddon.cmake
@@ -49,8 +49,8 @@
 
     get_target_property(PIA_LIB_DIR ${PIA_ADDON_NAME} LIBRARY_OUTPUT_DIRECTORY)
     install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${PIA_ADDON_NAME}.py
-            DESTINATION ${CMAKE_INSTALL_LIBDIR}/python${Python3_VERSION_MAJOR}.${Python3_VERSION_MINOR}/site-packages/)
-    install(FILES ${PIA_LIB_DIR}/${PIA_ADDON_LIBNAME}.so DESTINATION ${CMAKE_INSTALL_LIBDIR}/python${Python3_VERSION_MAJOR}.${Python3_VERSION_MINOR}/site-packages/)
+            DESTINATION ${CMAKE_INSTALL_PREFIX}/python/)
+    install(FILES ${PIA_LIB_DIR}/${PIA_ADDON_LIBNAME}.so DESTINATION ${CMAKE_INSTALL_PREFIX}/python/)
 
     else(CAN_BUILD)
         message("Compatible version of Swig found. NOT building ${PIA_ADDON_NAME}")
--- a/artdaq-database/SWIGBindings/python/conftool/CMakeLists.txt
+++ b/artdaq-database/SWIGBindings/python/conftool/CMakeLists.txt
@@ -1,4 +1,7 @@
 
+add_compile_definitions(ADFHICLCPP)
+set_property(SOURCE conftool.i PROPERTY COMPILE_DEFINITIONS ADFHICLCPP)
+
 create_python_addon ( ADDON_NAME "conftoolp"
 					  LIBRARIES
 					  artdaq-database::ConfigurationDB)
